web/src/compose_actions.js

```diff
@@ -259,7 +259,7 @@ export function start(msg_type, opts) {
     // If the user opens the compose box, types some text, and then clicks on a
     // different stream/topic, we want to keep the text in the compose box
     if (opts.content !== undefined) {
-        compose_state.message_content(opts.content);
+        compose_ui.insert_and_scroll_into_view(opts.content, $("textarea#compose-textarea"), true);
```

Comment:
So this doesn't clear existing content, is that a problem? I'm not sure if there ever will be something there.

Reply:
By passing true as the 3rd argument (`replace_all`), any existing content is cleared